website/src/site/apt/team.apt

```diff
@@ -69,7 +69,7 @@ Team
 *------------*--------------------------*---------------------------*
   jerryjung |       You Sun Jung       |         SK Telecom        |
 *------------*--------------------------*---------------------------*
-  dongjoon  |      Dongjoon Hyun       |                           |
+  dongjoon  |      Dongjoon Hyun       |         HortonWorks       |
```

Comment:
Congratulations for joining Hortonworks!
Though this information is irrelevant to this PR, I think creating a separate PR for the update as in #688 is an overshoot.
@markusweimer Do you think we need to open a new PR for this change?